jung-visualization/src/main/java/edu/uci/ics/jung/visualization/transform/shape/MagnifyImageLensSupport.java

```diff
@@ -45,8 +47,14 @@
           + "Mouse-Drag the Lens edge to resize it<p>"
           + "Ctrl+MouseWheel to change magnification</center></html>";
 
-  public MagnifyImageLensSupport(VisualizationViewer<V, E> vv, Dimension d) {
-    this(vv, new MagnifyShapeTransformer(d), new ModalLensGraphMouse());
+  public MagnifyImageLensSupport(VisualizationViewer<V, E> vv) {
```

Comment:
same comment as for corresponding `LayoutLensSupport` constructor: no `pickSupport`, should probably be calling other constructor